tests/test_email_handler.py

```diff
@@ -194,3 +187,15 @@ def test_dmarc_reply_quarantine(flask_client, dmarc_result):
         user_id=user.id, alert_type=ALERT_DMARC_FAILED_REPLY_PHASE
     ).all()
     assert len(alerts) == 1
+
+
+def test_add_alias_to_header_if_needed():
+    msg = EmailMessage()
+    create_new_user()
+    alias = Alias.first()
+
+    assert msg[headers.TO] is None
+
+    email_handler.add_alias_to_header_if_needed(msg, alias)
+
+    assert msg[headers.TO] == alias.email
```

Comment:
Nits: I'd add a test case for appending to the `To` header

```suggestion
    assert msg[headers.TO] == alias.email
    
    
def test_append_alias_to_header_if_needed():
    msg = EmailMessage()
    original_to = 'noone@nowhere.no'
    msg[headers.To] = original_to
    create_new_user()
    alias = Alias.first()

    email_handler.add_alias_to_header_if_needed(msg, alias)

    assert msg[headers.TO] == f"{original_to},{alias.email}"
```

Reply:
Good idea! Added in https://github.com/simple-login/app/pull/899/commits/01fd8809021e80f2f5613411dc60edbb55a959c0